cli/lib/kontena/cli/services/remove_command.rb

```diff
@@ -30,5 +42,17 @@ def execute
         end
       end
     end
+
+    def remove_instance
+      instance_name = "#{name}/#{instance}"
+      confirm_command("#{name}/#{instance}") unless forced?
+      service_instance = client.get("services/#{parse_service_id(name)}/instances")['instances'].find{ |i|
```

Comment:
If we're starting to make service instances a top-level API object... then should this command be `kontena service instance remove --instance 1 service`?